### PR TITLE
Return better error message when invalid yaml settings are used

### PR DESCRIPTION
**Pull request by sylvestor88**
So I have implemented a logic that reads parameters from cassandra.yaml and determines if the parameters in test yamls are valid/invalid. If invalid parameters are found an IllegalArgumentException is thrown which lists the invalid parameters.

### CODE REVIEW

File: test/validation/org/apache/cassandra/HarnessTest.java [hunk 1]
```diff
@@ -180,7 +180,7 @@ static URL getConfigURL(String yamlPath)
         }
     }
 
-    public String getTestName(String yamlPath)
+    static String getTestName(String yamlPath)
```
Comment (ptnapoleon) [inline test/validation/org/apache/cassandra/HarnessTest.java:183]: Why change this to static? Not that I disagree, but you don't appear to be using it in a static context anywhere in this PR

Comment (sylvestor88) [inline test/validation/org/apache/cassandra/HarnessTest.java:183]: yeah.. it got nothing to do with this PR.. I just noticed that while I was working on another function and just made it static.

Comment (ptnapoleon) [inline test/validation/org/apache/cassandra/HarnessTest.java:183]: Cool. It does make more sense to be static.

**Comment by sylvestor88** [comment]
updated PR with changing getTestName() to static

**Pull request merged**